From: Colin Ian King <colin.i.king@gmail.com>
Subject: lib/oid_registry.c: remove redundant assignment to variable num
Date: Mon, 17 Oct 2022 22:45:56 +0100

The variable num is being assigned a value that is never read, it is being
re-assigned a new value in both paths if an if-statement.  The assignment
is redundant and can be removed.

Cleans up clang scan build warning:
lib/oid_registry.c:149:3: warning: Value stored to 'num' is
never read [deadcode.DeadStores]

Link: https://lkml.kernel.org/r/20221017214556.863357-1-colin.i.king@gmail.com
Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/oid_registry.c |    1 -
 1 file changed, 1 deletion(-)

--- a/lib/oid_registry.c~lib-remove-redundant-assignment-to-variable-num
+++ a/lib/oid_registry.c
@@ -146,7 +146,6 @@ int sprint_oid(const void *data, size_t
 	bufsize -= count;
 
 	while (v < end) {
-		num = 0;
 		n = *v++;
 		if (!(n & 0x80)) {
 			num = n;
_
